### ISSUE TITLE
Typescript errors with non-experimental graphql

### ISSUE THREAD

**Issue by zachasme** [contrawork/graphql-helix#8]
Hi there!

I'm trying out helix (the ecosystem needed an alternative like this - thanks!), and had some trouble with typings:

```ts
node_modules/graphql-helix/dist/types.d.ts:1:24 - error TS2305: Module '"../../graphql"' has no exported member 'ExecutionPatchResult'.

1 import { DocumentNode, ExecutionPatchResult, ExecutionResult, GraphQLSchema, OperationDefinitionNode, ValidationRule } from "graphql";
                         ~~~~~~~~~~~~~~~~~~~~

Found 1 error.
```

After some digging I realized the `ExecutionPatchResult` interface [is only exported on the   `experimental-stream-defer` tag](https://github.com/graphql/graphql-js/blob/dc7f3cfffd104c3e994e1e2de0cfef5ed3a19825/src/execution/execute.d.ts#L82).

I understand helix supports both directives, so I guess it shouldn't be surprising.

However, I think helix could work better with a default installation of `graphql`. Would you accept a PR where the `ExecutionPatchResult` is copied instead of imported? Alternatively just a PR that documents it in the readme?

Again, thanks for a great alternative to apollo!

**Comment by danielrearden** [comment]
Thanks for the bug report @zachasme. This should be fixed now in `1.2.1`. Please give it a shot.

**Comment by zachasme** [comment]
Wow that was fast :+1: works like a charm!
